### PR TITLE
Improve CrashA detection

### PR DESCRIPTION
**Pull request by LewUwU**
Add check for invalid y position

### CODE REVIEW

**Comment by MWHunter** [comment]
I can't find the hard coded Y in the source code.

**Comment by MWHunter** [comment]
The only place I remember this value is in Verus's source code: https://github.com/DortDev/Verus-AntiCheat-SourceCode/blob/main/src/me/levansj01/verus/check/checks/crasher/ServerCrasherC.java

**Comment by LewUwU** [comment]
> The only place I remember this value is in Verus's source code: https://github.com/DortDev/Verus-AntiCheat-SourceCode/blob/main/src/me/levansj01/verus/check/checks/crasher/ServerCrasherC.java

This value is commonly used in anticrash projects

**Comment by amnoah** [comment]
Theoretically, a player could reach that Y value. This doesn't seem stable.

File: src/main/java/ac/grim/grimac/checks/impl/crash/CrashA.java [hunk 1]
```diff
@@ -21,7 +22,7 @@ public void onPacketReceive(PacketReceiveEvent event) {
         if (event.getPacketType() == PacketType.Play.Client.PLAYER_POSITION_AND_ROTATION) {
             WrapperPlayClientPlayerPositionAndRotation packet = new WrapperPlayClientPlayerPositionAndRotation(event);
 
-            if (Math.abs(packet.getPosition().getX()) > HARD_CODED_BORDER || Math.abs(packet.getPosition().getZ()) > HARD_CODED_BORDER) {
+            if (Math.abs(packet.getPosition().getX()) > HARD_CODED_BORDER || Math.abs(packet.getPosition().getZ()) > HARD_CODED_BORDER || Math.abs(packet.getPosition.getY()) > HARD_CODED_ILLEGAL_Y) {
```
Comment (Jaren8r) [inline src/main/java/ac/grim/grimac/checks/impl/crash/CrashA.java:25]:
```suggestion
            if (Math.abs(packet.getPosition().getX()) > HARD_CODED_BORDER || Math.abs(packet.getPosition().getZ()) > HARD_CODED_BORDER || Math.abs(packet.getPosition().getY()) > HARD_CODED_ILLEGAL_Y) {
```

**Review by Jaren8r — CHANGES_REQUESTED** [review]

**Pull request closed without merging**